### PR TITLE
ADEN-2789 Nielsen integration

### PR DESCRIPTION
**Pull request by fraszczakszymon**
Nielsen DCR integration

### CODE REVIEW

File: extensions/wikia/AnalyticsEngine/AnalyticsProviderNielsen.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?php
+
+class AnalyticsProviderNielsen implements iAnalyticsProvider {
+
+	private static $apid = 'T26086A07-C7FB-4124-A679-8AC404198BA7';
+	private static $clientId = 'Wikia';
+	private static $libraryUrl = 'http://secure-dcr-cert.imrworldwide.com/novms/js/2/ggcmb500.js';
+
+	function getSetupHtml( $params=array() ) {
```
Comment (nandy-andy) [inline extensions/wikia/AnalyticsEngine/AnalyticsProviderNielsen.php:9]: Not a short array notation fan you are, huh? (no Yoda emoicon :disappointed:)

**Comment by nandy-andy** [comment]
LGTM :sunglasses:

File: extensions/wikia/AnalyticsEngine/AnalyticsProviderNielsen.php (hunk 1 above)
Comment (fraszczakszymon) [inline extensions/wikia/AnalyticsEngine/AnalyticsProviderNielsen.php:9]: Meh :stuck_out_tongue_closed_eyes:. Just to make it consistent with other providers.

Comment (nandy-andy) [inline extensions/wikia/AnalyticsEngine/AnalyticsProviderNielsen.php:9]: OK, makes sense. Thanks for info!

**Comment by jacekjursza** [comment]
LGTM

**Pull request merged**